new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItSimpleDomainValidation.java

```diff
@@ -41,11 +44,50 @@ public void testCreatingDomain() {
         "Failed to create unique namespace due to ApiException");
     logger.info(String.format("Got a new namespace called %s", namespace));
 
+    // create a persistent volume claim
+    final String storageClassName = domainUID + "-weblogic-domain-storage-class";
+    final String pvcName = domainUID + "-pvc"; // name of the persistent volume claim
+    PersistentVolumeClaim persistentVolumeClaim = new PersistentVolumeClaim();
```

Comment:
Switch to standard types and then you'll also be able to chain all of these statements together.

Reply:
using standard types